.github/workflows/build.yml

```diff
@@ -0,0 +1,37 @@
+name: Test buildability
+
+on:
+  push:
+    branches: [ v2.2.6-for-tarantool-httpng ]
+  pull_request:
+    branches: [ v2.2.6-for-tarantool-httpng ]
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ['ubuntu-latest', 'macos-latest']
+        build-type: ['Release', 'Debug']
+    runs-on: ${{matrix.os}}
+
+    steps:
+    - name: Where we are?
+      run: |
+        uname -a; cat /etc/os-release || cat /etc/issue || python -m platform || true
+        gcc --version || true
```

Comment:
gcc is unused here, why do you need to show a version for it?

Reply:
This is a copy-paste from tarantool/httpng yml's which I have used for a lot of investigations.

gcc is used now. Thank you.